docs/topics/downloader-middleware.rst

```diff
@@ -745,6 +746,21 @@ Default: ``True``
 
 Whether the Compression middleware will be enabled.
 
+RETRY_HTTP_COMPRESSION_ERRORS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``False``
+
+Whether we should retry requests when an exception is raised while trying
+to decompress its response body.
+
+If an exception is raised while trying to decompress the response body,
+we store it in the request meta under the ``_http_compression_exc`` key and
+return the original ``scrapy.Response`` object. If the ``RetryMiddleware`` is
+enabled, it will check for this key and try to reprocess the ``scrapy.Request``
+if the captured exception is specified in the
+``RetryMiddleware.EXCEPTIONS_TO_RETRY`` tuple.
```

Comment:
I think we should either remove this “if …” statement from the documentation or expose `EXCEPTIONS_TO_RETRY` through a setting.

Reply:
I do agree with you. Although I prefer to expose this class attribute through a setting, this is out of the scope of this ticket. So I'm just removing the _if_.